cmd/carbonapi/main.go

```diff
@@ -66,12 +74,23 @@ func main() {
 				zap.Bool("pprof_enabled", config.Config.Expvar.PProfEnabled),
 			)
 
+			listener := config.Listener{
+				Address: config.Config.Expvar.Listen,
+			}
 			wg.Add(1)
-			go func() {
-				err = gracehttp.Serve(&http.Server{
-					Addr:    config.Config.Expvar.Listen,
+			go func(listen config.Listener) {
+				l := &net.ListenConfig{Control: helper.ReusePort}
+				s := &http.Server{
+					Addr:    listen.Address,
 					Handler: handler,
-				})
+				}
+				listener, err := l.Listen(context.Background(), "tcp", listen.Address)
```

Comment:
Can we resolve an address? E.g., localhost is always resolved to `127.0.0.1` + `::1`. Then function would create `s` per resolved adress